public/stylesheets/site/2.0/25-media-midsize.css

```diff
@@ -83,5 +95,5 @@ form.filters dl {
 /* 21 userstuff */
 
 #workskin {
-  margin: auto 5%;
-}
\ No newline at end of file
+  margin: auto 1.5%;
```

Comment:
Is this one going to cause drama from users, should we warn them in advance? Or have I misunderstood and it won't change that much because of something else?